include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,775 @@
+#ifndef _NBL_VIDEO_S_PHYSICAL_DEVICE_FEATURES_H_INCLUDED_
+#define _NBL_VIDEO_S_PHYSICAL_DEVICE_FEATURES_H_INCLUDED_
+
+namespace nbl::video
+{
+
+struct SPhysicalDeviceFeatures
+{
+    /* Vulkan 1.0 Core  */
+    bool robustBufferAccess = false;
+    bool fullDrawIndexUint32 = false;
+    bool imageCubeArray = false;
+    bool independentBlend = false;
+    bool geometryShader    = false;
+    bool tessellationShader = false;
+    //VkBool32    sampleRateShading;
+    bool dualSrcBlend = false;
+    bool logicOp = false;
+    bool multiDrawIndirect = false;
+    bool drawIndirectFirstInstance = false;
+    bool depthClamp = false;
+    bool depthBiasClamp = false;
+    bool fillModeNonSolid = false;
+    bool depthBounds = false;
+    bool wideLines = false;
+    bool largePoints = false;
+    bool alphaToOne = false;
+    bool multiViewport = false;
+    bool samplerAnisotropy = false;
+
+    // [DO NOT EXPOSE] these 3 don't make a difference, just shortcut from Querying support from PhysicalDevice
+    //VkBool32    textureCompressionETC2;
+    //VkBool32    textureCompressionASTC_LDR;
+    //VkBool32    textureCompressionBC;
+    
+    bool occlusionQueryPrecise = false;
+    //VkBool32    pipelineStatisticsQuery; [TODO]
+    bool vertexPipelineStoresAndAtomics = false;
+    bool fragmentStoresAndAtomics = false;
+    bool shaderTessellationAndGeometryPointSize = false;
+    bool shaderImageGatherExtended = false;
+    bool shaderStorageImageExtendedFormats = false;
+    bool shaderStorageImageMultisample = false;
+    bool shaderStorageImageReadWithoutFormat = false;
+    bool shaderStorageImageWriteWithoutFormat = false;
+    bool shaderUniformBufferArrayDynamicIndexing = false;
+    bool shaderSampledImageArrayDynamicIndexing = false;
+    bool shaderStorageBufferArrayDynamicIndexing = false;
+    bool shaderStorageImageArrayDynamicIndexing = false;
+    bool shaderClipDistance = false;
+    bool shaderCullDistance = false;
+    bool vertexAttributeDouble = false; // shaderFloat64
+    bool shaderInt64 = false;
+    bool shaderInt16 = false;
```

Comment:
whatever doesn't need API changes or special GL or common (validation) backend code, enable always and just report as a limit